release-tasks.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
 
-# enable echo mode
-set -x
+notify () {
+  FAILED_COMMAND="$(caller): ${BASH_COMMAND}" \
+    bundle exec rails runner "ReleasePhaseNotifier.ping_slack"
+}
+
+trap notify ERR
+
+# enable echo mode (-x) and exit on error (-e)
+set -ex
```

Comment:
@mstruve Let's please also add the `-E` option here. This will ensure that `ERR` traps get inherited by functions, command substitutions, and subshell environments. We don't need any of that in the current form, but it will make the script more robust.

Reply:
Done! Thanks!